web/.env.example

```
@@ -7,3 +7,15 @@ CMUX_FEEDBACK_RATE_LIMIT_ID=
 NEXT_PUBLIC_STACK_PROJECT_ID=
 NEXT_PUBLIC_STACK_PUBLISHABLE_CLIENT_KEY=
 STACK_SECRET_SERVER_KEY=
+
+# Cloud VM providers (manaflow-owned keys, no BYO). See scratch/vm-experiments for images.
+E2B_API_KEY=
+FREESTYLE_API_KEY=
+# Default sandbox template per provider. The E2B template is produced by
+# scratch/vm-experiments/images/build-e2b.ts; override via env when a newer build is published.
+E2B_SANDBOX_TEMPLATE=cmux-sandbox:v0-71a954b8e53b
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2B_SANDBOX_TEMPLATE key should go before the FREESTYLE_API_KEY key

(UnorderedKey)